config.py

```diff
@@ -196,9 +196,9 @@ def addBuilderModule(module):
 # internal web server (usually the html.WebStatus page) is visible. This
 # typically uses the port number set in the Waterfall 'status' entry, but
 # with an externally-visible host name which the buildbot cannot figure out
-# without some help.
+# without some help.  Do not add a '/' to the end. This will be added in code.
```

Comment:
"This will be added in code" is confusing. Please point to where it is added and perhaps say why it is not here.